lib/rules/declaration-block-no-redundant-longhand-properties/index.js

```diff
@@ -28,6 +28,42 @@ const meta = {
 
 /** @type {Map<string, (decls: Map<string, Declaration>) => (string | undefined)>} */
 const customResolvers = new Map([
+	[
+		'font-synthesis',
+		(decls) => {
+			const weight = decls.get('font-synthesis-weight')?.value.trim();
+			const style = decls.get('font-synthesis-style')?.value.trim();
+			const smallCaps = decls.get('font-synthesis-small-caps')?.value.trim();
+
+			const isValidFontSynthesisValue = (/** @type {string | undefined} */ s) =>
+				s === 'none' || s === 'auto';
```

Comment:
[suggest] Minor. The following seems a bit more readable to me:

```suggestion
			/** @type {(s: string | undefined) => boolean} */
			const isValidFontSynthesisValue = (s) => s === 'none' || s === 'auto';
```